src/syscalls/fs.rs

```diff
@@ -160,11 +159,14 @@ impl Filesystem {
 	}
 
 	/// Create new backing-fs at mountpoint mntpath
+	#[cfg(feature = "pci")]
 	pub fn mount(
 		&mut self,
 		mntpath: &str,
 		mntobj: Box<dyn PosixFileSystem + Send>,
 	) -> Result<(), ()> {
+		use alloc::borrow::ToOwned;
```

Comment:
Is this dead code removal?

Reply:
Yes. The import should be conditional. Having the import inside the function is more readable than having it conditionally at the head of the file, since this is the only use of the import.